pkg/operator/operator.go

```diff
@@ -139,7 +139,6 @@ func (pc *ProxyConfig) NoProxy() string {
 const (
 	resyncPeriod         = 15 * time.Minute
 	reconciliationPeriod = 5 * time.Minute
-	maxFailCount         = 2
```

Comment:
I prefer keep it as constant. no need to put it into the stack if it does not change ;)

Reply:
+1, we definitely should take measure to prevent this from changing.

Reply:
I think the performance/resource consumption differences between the two approaches are negligible here.

I think having it defined inside the function would make the code more readable (no need to jump into the variable to get the value + we centralize the comments) and less easier to be changed b mistake

Reply:
I think the relocation makes sense, since this is not used anywhere else. With the new scope I think making it a var is acceptable.